File: LollyJPlay/app/models/LollyForm.java
package models;

import java.util.List;
import java.util.Map;

import javax.persistence.Entity;

import com.zwstudio.lolly.domain.Language;

import play.data.validation.Constraints.Required;

@Entity
public class LollyForm {
	@Required
	public String word;
	public int selectedLangID;
	public String selectedDictName;
	public List<Language> langList;
	public Map<String, String> langMap;
}

File: LollySPlay/app/models/LollyForm.scala
package models

import scala.collection.immutable.Map

case class LollyForm(
  word: String = null,
  selectedLangID: Int = 0,
  selectedDictName: String = null,
  langList: Seq[Language] = null,
  langMap: Map[String, String] = null
)
